stracer/src/main/scala/com/evolutiongaming/random/ThreadLocalRandom.scala

```diff
@@ -1,20 +1,18 @@
 package com.evolutiongaming.random
 
-import cats.FlatMap
-import cats.effect.Clock
+import cats.effect.{Clock, Sync}
 import cats.syntax.all._
 import com.evolutiongaming.catshelper.{ThreadLocalOf, ThreadLocalRef}
 
 object ThreadLocalRandom {
 
-  def of[F[_]: FlatMap: Clock: ThreadLocalOf]: F[Random[F]] = {
-    val random = Random.State.fromClock[F]()
-    for {
-      random <- ThreadLocalOf[F].apply(random)
+  def of[F[_]: Sync: Clock: ThreadLocalOf]: F[Random[F]] = for {
```

Comment:
I merged this too fast… this is not backward compatible change, could you please fix?